tap_amazon_ads/streams/abstracts.py

```diff
@@ -200,20 +259,17 @@ def sync(
                         child.sync(state=state, transformer=transformer, parent_obj=record)
 
             state = self.write_bookmark(state, self.tap_stream_id, value=current_max_bookmark_date)
-            return counter.value
+            return counter.value, state
```

Comment:
added another comment about returning counters, 
ref: https://github.com/singer-io/singer-python/blob/master/singer/metrics.py#L134-L141

These are expected to log the record count after a certain interval and after the block completes execution.